pkg/addons/openebs/openebs.go

```diff
@@ -3,23 +3,26 @@
 package openebs
 
 import (
-	"context"
 	"fmt"
+	"io"
+	"os"
+	"path/filepath"
 	"strings"
 
+	"github.com/k0sproject/dig"
 	"github.com/replicatedhq/troubleshoot/pkg/apis/troubleshoot/v1beta2"
+	"github.com/sirupsen/logrus"
+	"github.com/urfave/cli/v2"
 	"golang.org/x/mod/semver"
-	"helm.sh/helm/v3/pkg/action"
-	"helm.sh/helm/v3/pkg/chart/loader"
-	"helm.sh/helm/v3/pkg/cli"
-	"helm.sh/helm/v3/pkg/release"
+	"gopkg.in/yaml.v2"
 
 	"github.com/replicatedhq/helmvm/pkg/addons/openebs/charts"
-	pb "github.com/replicatedhq/helmvm/pkg/progressbar"
+	"github.com/replicatedhq/helmvm/pkg/defaults"
 )
 
 const (
 	releaseName = "openebs"
+	appVersion  = "3.7.0"
```

Comment:
This shouldn't be hardcoded here (the same is valid for the admin console version). We have this version set also in the Makefile and now we need to remember updating in two places.